pkg/trtl/trtl.go

```diff
@@ -301,6 +347,7 @@ func (h *TrtlService) Iter(ctx context.Context, in *pb.IterRequest) (out *pb.Ite
 			log.Error().Err(err).Str("key", base64.RawURLEncoding.EncodeToString(iter.Key())).Msg("could not fetch object metadata")
 			return nil, status.Error(codes.FailedPrecondition, "database is in invalid state")
 		}
+		ns = object.Namespace
```

Comment:
@bbengfort is there a better place to retrieve Namespace from in Iter?

Reply:
We added `iter.Namespace()` to access the namespace that is being iterated on and you can use it outside of the `for` loop if that's easier.